server/pulp/server/webservices/views/content.py

```diff
@@ -518,14 +518,36 @@ def get(self, request):
             sources.append(d)
         return generate_json_response_with_pulp_encoder(sources)
 
+
+class ContentSourceCollectionActionView(View):
+    """
+    View for actions on all sources.
+    """
+
+    @staticmethod
+    def refresh(request):
```

Comment:
No reason really.  Just moved it as is.  But, I usually make methods like that static when PyCharm points out that it can be.